### ISSUE TITLE
Use Case: As a profile designer I want to use WKT for geo data to avoid ambiguity

### ISSUE THREAD

**Issue by ptsefton** [ResearchObject/ro-crate#266]
Assumption: should use schema:geo to point to an entity. But what should the entity look like?

Here's one way:

{
  "@type":  "Place",
  "name" : "Place 1",
  "geo": {"@id": "#point1"}
},

{
    "@id": "#point1",
      "@type": "PropertyValue",
      "propertyID": "http://www.opengis.net/ont/geosparql#wktLiteral",
      "value": "POINT(-76 -18)",
 }

**Comment by ptsefton** [comment]
Doing it like this will likely break our libraries which don't know how to deal with literals like this:

```json

 "geosparql:asWKT": {
  "@type": "http://www.opengis.net/ont/geosparql#wktLiteral",
"@value": "POINT(-76 -18)"
 },


```

**Comment by marc-portier** [comment]
Just checked how we ended up using these inside the LOD publications of https://MarineRegions.org/ 

From there 2 possible tips:
1. declare the type of these property once in the `@context`, so not to repeat it on every instance
    ```json
     "@context": { 
         "gsp": "http://www.opengis.net/ont/geosparql#"
         "gsp:asWKT": { 
              "@type": "gsp:wktLiteral"
    ```
2. and then on the values you use it also include a `<URI>` to depict the chosen CSR (projection system) at the start of the WKT literal:
    ```json
        "gsp:asWKT": "<http://www.opengis.net/def/crs/OGC/1.3/CRS84> POINT(-76 -18)"
    ```


you can check this online from the source for e.g.
```bash
curl -L --url "http://marineregions.org/mrgid/2196" -H "Accept: application/ld+json"  | python -m json.tool
```


HTH

**Comment by ptsefton** [comment]
@stain can you add this to the @context - I will add something to the text about how to use it for ContextualEntities

**Comment by stain** [comment]
Ping @rapw3k on checking on Earth Sciences approach

**Comment by stain** [comment]
Implemented by #288 that overrides #287.
